src/pages/ImportData.js

```diff
@@ -394,6 +413,8 @@ const ImportData = () => {
     [Utils.DEFAULT, () => ['Import Snapshot', `Snapshot ${snapshotName}`]]
   );
 
+  const isProtectedData = !!url && isProtected(url, format);
```

Comment:
Maybe it's just me but I would push this null check into `isProtected()` to make the calling code cleaner.

Also, now that we have both protected sources and destinations, I think the meaning `isProtected` could be confusing. Maybe `isProtectedSource` would be clearer?

Reply:
Good point!